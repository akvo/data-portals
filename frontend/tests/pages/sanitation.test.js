import React from 'react'
import { render } from '@testing-library/react'
import Sanitation from '../../pages/sanitation'

describe('Sanitation page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Sanitation />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
