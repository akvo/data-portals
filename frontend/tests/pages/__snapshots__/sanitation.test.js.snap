// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sanitation page matches snapshot 1`] = `
<DocumentFragment>
  <h1>
    Sanitation
  </h1>
</DocumentFragment>
`;
